src/modules/user/services/user.service.ts

```diff
@@ -22,9 +22,8 @@ export class UserServiceImpl extends UserService {
   }
 
   public override async findUser(
```

Comment:
you can make this method as private if you don't use it outside service, example:
```suggestion
  private async _findUser(
```